wit/world.wit

```diff
@@ -16,6 +16,7 @@ world http-trigger-rc20231018 {
 world platform {
   include wasi:cli/imports@0.2.0;
   import wasi:http/outgoing-handler@0.2.0;
+  import wasi:keyvalue/store@0.2.0-draft;
```

Comment:
We definitely need to decide how we want to handle introducing new imports. This will break any Spin runtime that does not have `wasi:keyvalue` support.

Reply:
It seems like we should be bumping the WIT package version here right? This is still a little scary as we haven't even done it with WASI 0.2.1 yet...

Reply:
Yes, this was one of my questions: https://github.com/fermyon/spin/pull/2486#issuecomment-2094965982.  Do we need a separate conversation about how to do this?

Reply:
> Do we need a separate conversation about how to do this?

Probably? I'm not sure that _anyone_ knows how best to handle this yet.